overlord/ifacestate/ifacemgr.go

```diff
@@ -122,3 +126,54 @@ func MockSecurityBackends(be []interfaces.SecurityBackend) func() {
 	backends.All = be
 	return func() { backends.All = old }
 }
+
+// SnapInterfaceState describes state of interfaces of a single snap.
+type SnapInterfaceState struct {
```

Comment:
I think this name promises too much for what it actually contains, do you expect it to have more responsibilites in the future?

Reply:
Well, who knew we would need this? :-)